USAGE.md

```diff
@@ -5,14 +5,21 @@
 	3. gradle shadowJar
 	4. java -jar ./build/libs/mike-all.jar (for GUI) or java -jar ./build/libs/mike-all.jar (for CLI)
 	
+## Switching between the CLI and GUI
+
+	1. Save progress to a file on your computer
+	2. Quit the current program
+	3. Run the desired interface (GUI or CLI)
+	4. Load the file that you saved earlier
+	
 # CLI commands
-	save <name>.json (optional <path>) - Save file to specific path or to current directory if no path is specified
-  	load <path> - Loads a file from an absolute path
+	save <name> (optional <path>) - Save progress to a specific path or to the current directory if no path is specified. Adding .json to the end of the file will save it as a JSON file.
```

Comment:
Save in the CLI has changed a little bit since we now use a File object directly instead of trying to do janky stuff with Path objects. Basically, the 'name' of the file would include the path itself, if they want to save it somewhere else.
For example:
save C:\Users\conor\Desktop\umlfile.json would save umlfile.json to my desktop, and that whole thing is the "name"
But
save umlfile.json also works, and would just save to my current working directory.

Both would save a file called umlfile.json, though. It just changes where it actually gets saved to.


TLDR: (optional <path>) can get removed

Reply:
Ah yea forgot about that. Could I make it like I did for load, where I say <filepath>? (The goal being filepath means it include both the path and the file name)

Reply:
Supposed to say < filepath > but I guess that makes it invisible lol